cli/src/main/scala/com/gu/pfi/cli/service/CliIngestionService.scala

```diff
@@ -72,7 +72,8 @@ class CliIngestionService(http: CliHttpClient)(implicit ec: ExecutionContext) ex
   }
 
   def deleteBlob(id: String): Attempt[Unit] = {
-    http.delete(s"/api/blobs/$id?deleteFolders=false&checkChildren=false").map { r =>
+
+    http.delete(s"/api/blobs/${URLEncoder.encode(id, "UTF-8")}?deleteFolders=false&checkChildren=false").map { r =>
```

Comment:
Do we definitely want `deleteFolders=false`? If so could we have a comment explaining why, or tweak the naming so it's clearer what it means?

If there was also a way to clarify the `checkChildren` terminology in some way that would be a bonus... `skipIfParent`? `skipIfHasChildren`? Or, `orphanChildren=true`?

In fact the more I think about it the more confused I am... what's the behaviour if we delete at zip file with `checkChildren=false`? It deletes the zip blob, but leaves its contents intact in neo4j and elasticsearch? ditto email archives like .pst

At this point I start re-reading https://docs.google.com/document/d/1obN21_zUF0iNKyLO60XGfvBtx2B2zaFxX1C--uhp2bA/edit# and my head explodes a bit 🤯